test/e2e/support/bootstrap.js

```diff
@@ -16,5 +21,20 @@ beforeAll( async () => {
 } );
 
 afterAll( async () => {
+	page.on( 'dialog', ( dialog ) => {
+		dialog.accept();
+	} );
+
+	await visitAdmin( 'edit.php' );
+
+	const bulkSelector = await page.$( '#bulk-action-selector-top' );
```

Comment:
I'm thinking I should add some comments to this explaining what it does, but the gist is that it opens the admin page and deletes every post on the page. This clears all posts and comments created by tests.

In theory, if we made so many posts in a single test that it spanned multiple pages we'd need to adjust this, but I started simple for now as that never happens.

Reply:
I'm wondering if we shouldn't perform it as `beforeAll` to support also the case where the previous run was interrupted by some random factor and cleanup could be finished.

Reply:
Well, the first time the tests are run the entire setup is reset: https://github.com/WordPress/gutenberg/pull/8041/files#diff-cc6d2a226eaf023f937fb5813d954327R10

I guess doing it in either place is fine though, so we can change it to `beforeEach` if you like. Actually, doing that might save us accepting the "Do you want to leave this page?" dialog so I'll give it a go 👍

Reply:
Just realised that `global.page` doesn't exist in the `beforeAll`.

We should really nix the globals in the test suite as well, they make it weird to reason about the state of tests and I don't think they're useful; but that's a separate issue I'll get into after this is fixed 😄